pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1994,2077 +1994,6 @@ func TestRunGeneralPredicates(t *testing.T) {
 	}
 }
 
-// TODO: Add test case for RequiredDuringSchedulingRequiredDuringExecution after it's implemented.
-func TestInterPodAffinity(t *testing.T) {
```

Comment:
This gets moved to pkg/scheduler/framework/plugins/interpodaffinity/interpod_affinity_test.go, right?

Reply:
yes, it is already there.